### PR TITLE
Overview changes

### PR DESCRIPTION
**Pull request by Nkozlare**
Fixed selector image bug.
Added navigation bar to page.
created state on App so everyone can have access to the current style.
I added a checkmark to the selected style
I added a review count to the overview as well as the get request to get the info, it will be a link tomorrow.
I finished the star component in overview as well as adding the dependency to package.json and yarn.lock

### CODE REVIEW

File: client/src/components/overview/GalleryView.jsx [hunk 1]
```diff
@@ -31,35 +34,11 @@ const StyledOverviewGrid = styled.div`
...
-});
-
-// Here is the axios request to receive receive the ratings of a product
-const retrieveRatingInfo = axios({
-  method: 'get',
-  url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/reviews/meta?product_id=37314',
-  headers: {
-    Authorization: API_KEY
-  }
-});
```
Comment (Nkozlare) [inline client/src/components/overview/GalleryView.jsx:62]: I just removed these functions and put them into the overview functional component

File: client/public/styles.css [hunk 2]
```diff
@@ -58,4 +58,62 @@ main {
   max-width: 840px;
   padding: 16px;
   margin: auto;
+}
+
+.topnav {
```
Comment (nicholaspix) [inline client/public/styles.css:63]: Well done on the css for the nav bar, looks clean and functional.

File: client/src/app.jsx [hunk 3]
```diff
@@ -1,17 +1,54 @@
...
+      behavior: "smooth"
+    });
+  }
 
-  return  <div >
-           <h1>Front End Capstone Avatar Project</h1>
-              <Overview />
+  return  (
+          <div>
+            {/* this is the start of the navbar */}
```
Comment (nicholaspix) [inline client/src/app.jsx:20]: This makes sense to me, looks good. I'm sure handleScroll is complicated in the background though.

File: client/src/components/overview/GalleryView.jsx [hunk 4]
```diff
@@ -81,7 +60,9 @@ function findAverageRating(ratings)  {
 
 
 // This is the actual functional component
-function Overview(props) {
+function Overview({ currentStyleId, setCurrentStyleId }) {
```
Comment (nicholaspix) [inline client/src/components/overview/GalleryView.jsx:63]: What does 'sku' mean?

File: client/src/components/overview/GalleryView.jsx [hunk 5]
```diff
@@ -179,18 +199,36 @@ function Overview(props) {
       });
     }
   });
-
+  /* find(obj => obj.style_id === currentStyleId) */
   if (!expanded) {
     return (
       <StyledOverviewGrid>
```
Comment (nicholaspix) [inline client/src/components/overview/GalleryView.jsx:205]: This code looks much cleaner than before, I should look to do something similar.

File: client/src/components/overview/overview-components/Stars.jsx [hunk 6]
```diff
@@ -1,15 +1,25 @@
...
 margin-top: 50px
 `;
 
-export default function OverviewStars({ stars }) {
+export default function OverviewStars({ stars, reviewCount }) {
   return (
-    <StyledOverviewStars>
-        <p>&#9733;&#9733;&#9733;&#9733;&#9733;</p>
+    <StyledOverviewStars data-testid="stars">
+        <StarRatings
```
Comment (nicholaspix) [inline client/src/components/overview/overview-components/Stars.jsx:14]: what is parseFloat?

**Review by nicholaspix — APPROVED** [review]

File: client/src/components/overview/GalleryView.jsx (hunk 4 above)
Comment (Nkozlare) [reply client/src/components/overview/GalleryView.jsx:63]: That is the technical term for a product id. But I should change it to match the api data.

File: client/src/components/overview/overview-components/Stars.jsx (hunk 6 above)
Comment (Nkozlare) [reply client/src/components/overview/overview-components/Stars.jsx:14]: Parse float leaves the decimals in. ParseInt removes everything after the dot.

**Pull request merged**